src/Languages.re

```diff
@@ -0,0 +1,24 @@
+/*
+     Languages.re
+ */
+
+type t =
+  | Json
+  | C
+  | Cpp
+  | Python
+  | Javascript
+  | Typescript
+  | Tsx;
+
+let getLanguageFromScope = scope =>
+  switch (scope) {
+  | "source.json" => Some(Json)
+  | "source.c" => Some(C)
+  | "source.cpp" => Some(Cpp)
+  | "source.python" => Some(Python)
+  | "source.js" => Some(Javascript)
+  | "source.ts" => Some(Typescript)
+  | "source.tsx" => Some(Tsx)
+  | _ => None
+  };
```

Comment:
Is this the best way to do this?

I.e. right now you give a scope and get back an option then action on that, and assuming there is a language, you can use that to request the parser/scope converter. Wasn't sure if there was a cleaner way, that also keeps one source of truth for the supported languages, so changing this list enforces a parser and scope converter are added.

Reply:
I think this is reasonable - although we could also consider an API that gives a `parser/scope converter` combo for a given scope. But for now I think this OK - as we use it, we could see if it could be streamlined